Gemfile.lock

```diff
@@ -239,28 +252,26 @@ GEM
     sawyer (0.8.2)
       addressable (>= 2.3.5)
       faraday (> 0.8, < 2.0)
-    simpleidn (0.1.1)
+    simpleidn (0.2.1)
       unf (~> 0.1.4)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thread_safe (0.3.6)
     typhoeus (1.4.0)
       ethon (>= 0.9.0)
-    tzinfo (1.2.7)
+    tzinfo (1.2.9)
       thread_safe (~> 0.1)
-    tzinfo-data (1.2020.1)
+    tzinfo-data (1.2021.5)
       tzinfo (>= 1.0.0)
     unf (0.1.4)
       unf_ext
-    unf_ext (0.0.7.7)
-    unf_ext (0.0.7.7-x64-mingw32)
-    unicode-display_width (1.7.0)
+    unf_ext (0.0.8)
+    unicode-display_width (1.8.0)
     wdm (0.1.1)
-    zeitwerk (2.4.0)
+    zeitwerk (2.5.1)
 
 PLATFORMS
-  ruby
-  x64-mingw32
```

Comment:
Should we restore these lines? Not sure if they're autogenerated or not, but we do want this to be supported on multiple platforms. The darwin one is just for macOS.

Reply:
hmm yeah just reading about that.

Reply:
https://www.moncefbelyamani.com/understanding-the-gemfile-lock-file/